adafruit_bme680.py

```diff
@@ -381,14 +388,14 @@ def _read_calibration(self):
         self._heat_val = self._read_byte(0x00)
         self._sw_err = (self._read_byte(0x04) & 0xF0) / 16
 
-    def _read_byte(self, register):
+    def _read_byte(self, register: int) -> int:
         """Read a byte register value and return it"""
         return self._read(register, 1)[0]
 
-    def _read(self, register, length):
+    def _read(self, register: int, length: int) -> None:
         raise NotImplementedError()
 
-    def _write(self, register, values):
+    def _write(self, register: int, values) -> None:
```

Comment:
The type can also be blueprinted for values:

```suggestion
    def _write(self, register: int, values: ReadableBuffer) -> None:
```